src/govee_ble/parser.py

```diff
@@ -87,13 +87,13 @@ def _process_mfr_data(
         service_uuids: list[str],
     ) -> None:
         """Parser for Govee sensors."""
-        _LOGGER.debug("Parsing Govee sensor: %s %s", mgr_id, data)
+        _LOGGER.debug("Parsing Govee sensor: %s %s", mgr_id, data.hex())
```

Comment:
Would you please adjust it to log both?  The binary data makes more sense to me because I'm used to looking at it